install_ffmpeg.sh

```diff
@@ -205,7 +205,7 @@ fi
 
 if [[ $BUILD_TAGS == *"debug-video"* ]]; then
   echo "video debug mode, building ffmpeg with tools, debug info and additional capabilities for running tests"
-  DEV_FFMPEG_FLAGS="--enable-muxer=md5,flv --enable-demuxer=hls --enable-filter=ssim,tinterlace --enable-encoder=wrapped_avframe,pcm_s16le "
```

Comment:
Why remove this one? Are those only extra from the non-dev ones?

Reply:
Yes, the base configuration includes flv now, so specifying it (again) with the dev flags isn't necessary (trying to keep things tidy)